cloudbuild.yaml

```diff
@@ -0,0 +1,68 @@
+timeout: 3600s # 1 hour
+logsBucket: 'gs://oppia-cloudbuild-logs'
+steps:
+  - id: build-oppia-base
+    name: gcr.io/kaniko-project/executor:latest
+    wait_for: ["-"]
+    args: 
+    - --dockerfile=Dockerfile.base
+    - --destination=gcr.io/oppia-jenkins-test-server/oppia-base:$SHORT_SHA
+    - --cache=true
+    - --cache-ttl=24h
```

Comment:
Why this time? I'm just curious to know what your thought process was. It's also totally fine if this is arbitrary!

Reply:
It's arbitrary. We can set the cache expiration to whatever we want.